### ISSUE TITLE
Helper function

### ISSUE THREAD

**Issue by asafyish** [clarkmoody/rust-bech32#3]
Hi,

Lets say I have a base32 string 'nbswy3dpo5xxe3de', I was expecting this library to support it as input or at least provide with an helper function to do so, but it seems I need a pre-step to convert the string to a special vec with integers ranging from 0 to 31.

Does it makes sense to add this helper function ?

**Comment by clarkmoody** [comment]
Try using 
```rust
let decode_result = Bech32::from_string("nbswy3dpo5xxe3de".to_string())
```

**Comment by asafyish** [comment]
That's for decoding, which I am able to do.
I am having trouble encoding,
Had to use convert_bits from https://github.com/petertodd/rust-bech32/blob/master/src/wit_prog.rs,
something like:

```rust
    let public_key = "d1jprv3fexqq4v34".to_string().into_bytes();
    let public_key = convert_bits(public_key, 8, 5, true).unwrap();
    println!("public key {:?}", public_key);

    let b = Bech32 {
        hrp: "poc".to_string(), 
        data: public_key
    };
    let encoded = b.to_string().unwrap();
```

maybe convert_bits should be provided in this lib ?

**Comment by clarkmoody** [comment]
I wrapped that functionality in the [bitcoin-bech32](https://crates.io/crates/bitcoin-bech32) create, though `convert_bits` is private there. It does make sense to have `convert_bits` live within this module. I'll work on porting it over from `bitcoin-bech32`.

**Comment by clarkmoody** [comment]
Resolved with #9
